src/statemachine.rs

```diff
@@ -532,7 +548,7 @@ impl StateMachine {
                 let mut skip_uv = false;
                 while alive() {
                     let mut permissions = PinUvAuthTokenPermission::MakeCredential;
-                    if makecred.needs_pre_flight(&dev) {
+                    if needs_pre_flight(&dev, &makecred.exclude_list) {
```

Comment:
I had it because the spec says:
> Note: The platform SHOULD request only the permissions absolutely necessary.

 But I think you are right, we can probably just always request MC and GA here.